### PR TITLE
pinning for windows platform.

### PR DESCRIPTION
**Pull request by TingluoHuang**

### CODE REVIEW

**Comment by thechrisjohnson** [comment]
Looks good.

File: Tasks/ANT/task.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 12
+        "Patch": 13
```
Comment (bryanmacfarlane) [inline Tasks/ANT/task.json]: Is 91 also patch 13?

**Comment by bryanmacfarlane** [comment]
We have to do an RM patch of these tasks?

**Comment by TingluoHuang** [comment]
Is the fix already in master? Yes
What testing was done? Verify the change won't break existing M90 agent, and it will help M91 agent figure out prefer handler base on platform.
Does this fix change the AT/DT contract? No
Does this fix include any database servicing changes? No

File: Tasks/ANT/task.json (hunk 1 above)
Comment (TingluoHuang) [inline Tasks/ANT/task.json]: all M91 tasks has higher version

**Pull request merged**